src/applications/personalization/dashboard/components/CTALink.jsx

```diff
@@ -18,14 +15,7 @@ const CTALink = ({ ariaLabel, className, href, text, onClick, newTab }) => {
       onClick={onClick || undefined}
       className={classNames}
     >
-      {`${firstWords} `}
```

Comment:
this weird "nowrap" thing was done to make sure the chevron icon was never orphaned from the link text its associated with. but since we no longer need to ever show the chevron icon in this component, I'm removing all of the special handling. It all still exists in the IconCTALink, though, in case we ever need to refer to it.